kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/ProxyReconcilerIT.java

```diff
@@ -63,28 +84,34 @@ void testCreate() {
     KafkaProxy doCreate() {
         final var cr = extension.create(testResource());
 
-        await().untilAsserted(() -> {
+        Awaitility.waitAtMost(Duration.ofSeconds(30)).logging().alias("Secret as expected").untilAsserted(() -> {
```

Comment:
nit: This await has `.logging` but none of the others do. Should we add it to all or remove it from here